apps/api/src/bank-transactions/bank-transactions.service.ts

```diff
@@ -65,6 +67,11 @@ export class BankTransactionsService {
       },
       skip: pageIndex && pageSize ? pageIndex * pageSize : undefined,
       take: pageSize ? pageSize : undefined,
+      orderBy: [
+        {
+          [sortBy as string]: sortOrder,
+        },
+      ],
```

Comment:
might be good to set a default sorting if the param is not sent by frontend, something like:
orderBy: [sortBy ? { [sortBy]: sortOrder ? sortOrder : 'desc' } : { transactionDate: 'desc' }],

Reply:
Implemented :+1: